=== modified file 'tools/indicator-loader.c'
--- tools/indicator-loader.c	2011-07-06 21:33:09 +0000
+++ tools/indicator-loader.c	2011-07-07 13:25:11 +0000
@@ -44,7 +44,11 @@
 	g_debug("Signal: Entry Added");
 
 	GtkWidget * menuitem = gtk_menu_item_new();
+#if GTK_CHECK_VERSION(3,0,0)
 	GtkWidget * hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 3);
+#else
+	GtkWidget * hbox = gtk_hbox_new(FALSE, 3);
+#endif
 
 	if (entry->image != NULL) {
 		gtk_box_pack_start(GTK_BOX(hbox), GTK_WIDGET(entry->image), FALSE, FALSE, 0);

